docs-src/index.md.template

```diff
@@ -0,0 +1,24 @@
+<p align="center">
+  <img alt="Layer0 Logo" src="/static/logo_rs.png" >
+</p>
+
+<center>
+# Build, Manage, and Deploy Your Application
+</center>
+
+
+## Meet Layer0
+Layer0 is a framework that helps you deploy web applications to the cloud with minimal fuss. Using a simple command line interface (CLI), you can manage the entire life cycle of your application without having to focus on infrastructure.
+
+Ready to learn more about Layer0? See our [introduction page](intro.md) to learn about some important concepts. When you're ready to get started, take a look at the [installation page](setup/install.md) for information about setting up Layer0.
+
+## Download
+| Download **v0.10.4** | | |
```

Comment:
why is `v0.10.4` hardcoded in the template?

Reply:
Hmmm 😕 I need to fix that. I thought I checked this file out before I pushed it.